test/unit/api_spec.js

```diff
@@ -1102,6 +1102,19 @@ describe("api", function () {
       await loadingTask.destroy();
     });
 
+    it("gets outline with non-displayable chars", async function () {
```

Comment:
Do we need this specific unit-test, given that a more general one is added below?

Reply:
I let the test to avoid any future api breakages (if the test was here I wouldn't have written the first version of this patch).
But if you think it's useless I'm fine to remove it.

Reply:
Let's just keep it then :-)